pkg/micropython/doc.txt

```diff
@@ -0,0 +1,74 @@
+/**
+ * @defgroup pkg_micropython MicroPython RIOT port
+ * @ingroup  pkg
+ * @brief    MicroPython - Python for microcontrollers
+ *
+ * # MicroPython RIOT package
+ *
+ * "MicroPython is a lean and efficient implementation of the Python 3
+ *  programming language that includes a small subset of the Python standard
+ *  library and is optimised to run on microcontrollers and in constrained
+ *  environments."
+ *
+ * @see https://micropython.org
+ *
+ * ## Status
+ *
+ * MicroPython on RIOT has to be considered experimental. While the basic
+ * interpreter works fairly well on native and Cortex-M, it has not seen much
+ * testing.
+ *
+ * ## Implementation details
+ *
+ * The RIOT port of MicroPython currently resides in a fork at
+ * https://github.com/kaspar030/micropython. It is based on Micropython's
```

Comment:
This should point to the [`add_riot_port`](https://github.com/kaspar030/micropython/tree/add_riot_port) branch

Reply:
yup, fixed